src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
```

Comment:
In the backend PR there was a concert that the failure might be temporary. If we cache the URL the image won't come till the next song.